tinker/static/assets/css/tinker.css

```diff
@@ -691,4 +710,32 @@ label.checkbox {
   color: #0f2c52; }
 
 .admin-user-bar > p > a:hover {
-  color: #337ab7; }
\ No newline at end of file
+  color: #337ab7; }
```

Comment:
put the } on a new line, for consistency sake

Reply:
Fixed this.